monitor/api/models/schema.py

```diff
@@ -2,20 +2,30 @@
 from pydantic import BaseModel, validator
 
 
-class ModelSchema(BaseModel):
+class NodeSchema(BaseModel):
     name: str
     unique_id: str
-    database_name: str
-    schema_name: str
-    table_name: str
     owners: Optional[str]
     tags: Optional[str]
     package_name: str
```

Comment:
I think package name is also optional

Reply:
I am not sure, but will change for just in case